JournalCoreData/View Controllers/EntryDetailViewController.swift

```diff
@@ -16,7 +16,7 @@ class EntryDetailViewController: UIViewController {
     
     @IBAction func saveEntry(_ sender: Any) {
         
-        guard let title = titleTextField.text,
+        guard let title = titleTextField.text, !title.isEmpty,
```

Comment:
Good catch! Maybe dell the user that they need a title, or disable the save button until the title gets filled in, to help them realize this restriction.